src/twisted/python/test/test_release.py

```diff
@@ -1400,7 +1412,7 @@ def test_remove(self):
         """
         reposDir = self.makeRepository(self.tmpDir)
         testFile = reposDir.child('some-file')
-        testFile.setContent("something")
```

Comment:
The only type of development beating it is probably backpressure and connection control in HTTP ;-) (heyyy @Lukasa ).

I'm inclined to agree, as I can't disagree with your impressions here; there is a certain mind-numbing tedium to both writing and reviewing py3 porting code, and that does result in some mistakes.  Perhaps addressing this feedback will be a good opportunity for someone (not necessarily @rodrigc, any committer can address it) to get better at identifying the appropriate text or bytes type to use in various cases.